db/migrate/20151027134345_convert_classification_subject_ids_to_join_table.rb

```diff
@@ -0,0 +1,12 @@
+class ConvertClassificationSubjectIdsToJoinTable < ActiveRecord::Migration
+  def change
+    create_table :classification_subjects, id: false do |t|
+      t.integer :classification_id, null: false
+      t.integer :subject_id, null: false
+    end
+
+    add_index :classification_subjects, [:classification_id, :subject_id], unique: true, name: 'classification_subjects_pk'
```

Comment:
I guess we should have indexes on both columns seperately, for lookups of all subjects for a classification and vice versa. Good catch.